web-ncer/app/services/request/ActionGraph.scala

```diff
@@ -1,101 +0,0 @@
-package services.request
-
-import models.{Action, BatchRequestState, Role, RequestAction}
-import org.incal.core.util.GroupMapList
-
-object ActionGraph {
-
-  val needsDescription = true
-
-  // TODO: move to custom.conf (or a dedicated conf file)
-
-  private val actions: Traversable[Action] = Seq(
-    Action(
-      BatchRequestState.Created,
-      RequestAction.Submit,
-      BatchRequestState.SentForApproval,
-      Role.Requester,
-      Role.Committee,
-      Seq(Role.Requester),
-      needsDescription
-    ),
-
-    Action(
-      BatchRequestState.SentForApproval,
-      RequestAction.Approve,
-      BatchRequestState.Approved,
-      Role.Committee,
-      Role.Owner,
-      Seq(Role.Requester, Role.Committee)
-    ),
-
-    Action(
-      BatchRequestState.SentForApproval,
-      RequestAction.Reject,
-      BatchRequestState.Rejected,
-      Role.Committee,
-      Role.Requester,
-      Seq(Role.Committee),
-      needsDescription
-    ),
-
-    Action(
-      BatchRequestState.Approved,
-      RequestAction.Approve,
-      BatchRequestState.OwnerAcknowledged,
-      Role.Owner,
-      Role.Owner,
-      Seq(Role.Requester)
-    ),
-
-    Action(
-      BatchRequestState.Approved,
-      RequestAction.NotAvailable,
```

Comment:
a lot of handy stuff, such as the workflow/approval state machine, has been removed (since not needed anymore)... maybe we can extract and archive it somewhere